[2022-08-12 11:48:21,738] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.creating_raw_data_dir manual__2022-08-12T11:48:20.133107+00:00 [queued]>
[2022-08-12 11:48:21,750] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.creating_raw_data_dir manual__2022-08-12T11:48:20.133107+00:00 [queued]>
[2022-08-12 11:48:21,751] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-12 11:48:21,752] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-08-12 11:48:21,753] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-12 11:48:21,771] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): creating_raw_data_dir> on 2022-08-12 11:48:20.133107+00:00
[2022-08-12 11:48:21,778] {standard_task_runner.py:52} INFO - Started process 1099 to run task
[2022-08-12 11:48:21,783] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_Raízen', 'creating_raw_data_dir', 'manual__2022-08-12T11:48:20.133107+00:00', '--job-id', '175', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmpmrbmagqy', '--error-file', '/tmp/tmp6aq7n_xw']
[2022-08-12 11:48:21,784] {standard_task_runner.py:80} INFO - Job 175: Subtask creating_raw_data_dir
[2022-08-12 11:48:21,881] {task_command.py:369} INFO - Running <TaskInstance: ETL_Raízen.creating_raw_data_dir manual__2022-08-12T11:48:20.133107+00:00 [running]> on host 3c4d8e0d9450
[2022-08-12 11:48:21,978] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL_Raízen
AIRFLOW_CTX_TASK_ID=creating_raw_data_dir
AIRFLOW_CTX_EXECUTION_DATE=2022-08-12T11:48:20.133107+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-12T11:48:20.133107+00:00
[2022-08-12 11:48:21,980] {ETL_Raizen.py:27} INFO - Dir already exists
[2022-08-12 11:48:21,980] {python.py:173} INFO - Done. Returned value was: None
[2022-08-12 11:48:21,997] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_Raízen, task_id=creating_raw_data_dir, execution_date=20220812T114820, start_date=20220812T114821, end_date=20220812T114821
[2022-08-12 11:48:22,036] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-12 11:48:22,106] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
